src/coreclr/tools/ILVerification/ILVerification.csproj

```diff
@@ -16,5 +16,6 @@
 
   <ItemGroup>
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
+    <PackageReference Include="Microsoft.Bcl.Memory" Version="$(MicrosoftBclMemoryVersion)" />
```

Comment:
What is this dependency needed for? Is it worth it?

Reply:
I believe I added this so I could use collection expression patterns. I don't think any of the types are actually used in the final compilation, but Roslyn requires `System.Index` and `System.Range` to be resolvable in the compilation to use any collection patterns.

Somewhat related: Why is `ILVerification.csproj` targeting netstandard2.0? Is there a reason we haven't moved it forward to NetCoreAppToolCurrent or NetCoreAppCurrent?